apps/files/src/components/Collaborators/SharedFilesList.vue

```diff
@@ -1,12 +1,65 @@
+<template>
+  <file-list :fileData="fileData" id="files-list" :loadingFolder="loadingFolder" :actions="actions" :compactMode="_sidebarOpen"
+    :isActionEnabled="isActionEnabled">
+    <template #headerColumns>
+      <oc-table-cell type="head" class="uk-text-truncate" v-translate>Name</oc-table-cell>
+      <oc-table-cell shrink type="head" class="uk-text-nowrap" v-text="sharedCellTitle" />
+      <oc-table-cell
+        v-if="$route.name === 'files-shared-with-me'"
+        shrink
+        type="head"
+        class="uk-text-nowrap"
+        v-translate
+      >
+        Status
+      </oc-table-cell>
+      <oc-table-cell shrink type="head" class="uk-text-nowrap" v-translate>Share time</oc-table-cell>
+    </template>
+    <template v-slot:rowColumns="{ item }">
```

Comment:
Get rid pls of `v-slot`. https://vuejs.org/v2/style-guide/#Directive-shorthands-strongly-recommended